content/templates.xqm

```diff
@@ -342,8 +342,12 @@ declare %private function templates:cast($values as item()*, $targetType as xs:s
             switch ($targetType)
                 case "xs:string" return
                     string($value)
-                case "xs:integer" case "xs:int" case "xs:long" return
+                case "xs:integer" return
                     xs:integer($value)
+                case "xs:int" return
+        		    xs:int($value)
+		        case "xs:long" return
+		            xs:long($value)
```

Comment:
The whitespace here looks strange in the diff view. Looks like some tabs were introduced, whereas the file only uses spaces.

Reply:
If we agree to merging this PR I would fix that.

Reply:
Sounds good to me.